tools/fds/README.txt

```diff
@@ -41,3 +42,8 @@ Examples:
 
       $ fast-discovery-server -i 1 -l 172.30.144.1 -p 12345 -b
```

Comment:
I know these are previous typos but I think we should fix them:

```suggestion
      $ fast-discovery-server -i 4 -l 172.30.144.1 -p 12345 -b
```

Also, the previous example is also mistaken, both the server ID as the first UDP port are not correct:

```
      $ fast-discovery-server -i 3 -l 192.168.36.34 -p 8783 -l 172.20.96.1 -p 51083
```

Reply:
Amazing catch!